lib/rollbar/request_data_extractor.rb

```diff
@@ -201,8 +201,8 @@ def rollbar_route_params(env)
       end
     end
 
-    def rollbar_request_session(rack_req)
-      session = rack_req.session
```

Comment:
Okay so the way I read the Rack::Request code is that this line is roughly equivalent to

```ruby
session = env.fetch('rack.session', nil)
if session.nil?
    env['rack.session'] = {}
    session = env['rack.session']
end
```

In other words, Rack::Request modifies the underlying `env` by setting the `rack.session` header if it is missing. So using the Rack::Request to pull the session introduces an ordering issue into the middleware.